solver/src/solver/paraswap_solver.rs

```diff
@@ -46,10 +47,10 @@ impl ParaswapSolver {
         web3: Web3,
         settlement_contract: GPv2Settlement,
         token_info: Arc<dyn TokenInfoFetching>,
-        slippage_bps: usize,
+        slippage_bps: u32,
```

Comment:
Changed to `u32` since in theory there are `usize` values that aren't representable in double precision floating point numbers, and many JSON parsers use those by default.